src/easier68k/core/opcodes/move.py

```diff
@@ -0,0 +1,216 @@
+from ...core.enum.ea_mode import EAMode
+from ...core.enum.op_size import MoveSize
+from ...core.enum.ea_mode_bin import EAModeBinary
+from ...simulator.m68k import M68K
+from ...core.opcodes.opcode import Opcode
+
+
+class Move(Opcode):
+    pass
+
+
+class Move(Opcode):
+    # Allowed values: nothing, or some combination of B, W, and L (for byte, word, and long)
+    # For example, MOVE would have 'BWL' because it can operate on any size of data, while MOVEA would have 'WL' because
+    # it can't operate on byte-sized data
+    allowed_sizes = 'BWL'
+
+    def __init__(self, command: str, parameters: str):
+        """
+        >>> str(Move('MOVE.B', '-(A0), D1'))
+        'Move command: Size B, src Mode: 4, Data: 0, dest Mode: 0, Data: 1'
+
+        >>> str(Move('MOVE.L', 'D3, (A0)'))
+        'Move command: Size L, src Mode: 0, Data: 3, dest Mode: 2, Data: 0'
+
+        >>> Move('MOVE.W', 'D3, A3')
+        Traceback (most recent call last):
+        ...
+        AssertionError
+
+        :param command:
+        :param parameters:
+        """
+        parts = command.split('.')  # Split the command by period to get the size of the command
```

Comment:
Do you ever assert that command is "MOVE" (also need to deal with cases).

Could maybe help to make a static helper method that does this so we aren't copy pasting code a lot

Reply:
I don't, I'll add that. And agreed